lib/galaxy/datatypes/sequence.py

```diff
@@ -36,8 +35,7 @@
 from galaxy.util.image_util import check_image_type
 from . import data
 
-if sys.version_info > (3,):
-    long = int
+long = int
```

Comment:
Should we replace `long` with `int` in the file instead?

Reply:
I went ahead and did this across the codebase; good idea.